lib_expense/src/test/java/com/example/expenselogger/lib_expense/domain/usecase/GetExpenseTest.kt

```diff
@@ -23,4 +26,13 @@ public class GetExpenseTest {
             val expectedExpense = getExpense.invoke(expense.id)
             assertThat(expectedExpense).isEqualTo(expense)
         }
+
+    @Test
+    public fun `verify that getExpense usecase calls expenseContract`(): Unit = runBlockingTest {
+        val expense = DummyData.expense
+        val mockExpenseContract = mock(ExpenseContract::class.java)
+        val mockGetExpense = GetExpense(mockExpenseContract, TestAsyncExecutor())
+        mockGetExpense.invoke(expense.id)
+        verify(mockExpenseContract).getExpense(expense.id)
```

Comment:
For use cases that return data, you need to verify that the correct data is returned for any given input

Reply:
Okay, I will verify that. Thank you